examples/librados/Makefile

```diff
@@ -1,11 +1,11 @@
 
-CXX?=g++
-CXX_FLAGS?=-std=c++11 -Wno-unused-parameter -Wall -Wextra -Werror -g
+CXX?=g++-11
```

Comment:
Why selecting a specific version? I'd rather use 12 if available. Or am I misunderstanding?

Reply:
We use GCC 11 from ppa:ubuntu-toolchain-r/test (added in this commit, otherwise newest version is 9). GCC 12 is not available yet https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test.

Reply:
I see. Were it CMake, there are better ways to say "11 or better". But not for makefile.